graylog2-web-interface/src/components/bootstrap/BootstrapModalForm.jsx

```diff
@@ -67,7 +67,10 @@ class BootstrapModalForm extends Component {
     );
 
     return (
-      <BootstrapModalWrapper ref="modal" onOpen={this.props.onModalOpen} onClose={this.props.onModalClose}>
+      <BootstrapModalWrapper ref="modal"
+                             onOpen={this.props.onModalOpen}
+                             onClose={this.props.onModalClose}
+                             onCancel={this._onModalCancel}>
```

Comment:
After 6ce4be2e4dfcec03333fbcd9ada9457e78de23e2, things are starting to get confusing again. Shouldn't we rename the original `onModalClose` to `onCancel`, and just keep a single callback?

The thing is, if I didn't miss anything, closing the modal and cancelling is doing the same thing right now: call the `onCancel` callback if available, and then call `close` on the `BootstrapModalWrapper`, that will eventually call the `onModalClose` prop. Well, when the modal is closed with escape, clicking outside, or pressing the "x", we call `onModalClose` twice.